### ISSUE TITLE
Images that link to their own media file link to the source network instead when cloned to a new network

### ISSUE THREAD

**Issue by mcgratay** [pressbooks/pressbooks#1835]
### Prerequisites

* [x] Can you reproduce the behavior you wish to address with [all other plugins deactivated](http://codex.wordpress.org/Multisite_Network_Administration#Plugins)?
* [x] Are you running the [latest version of Pressbooks](http://github.com/pressbooks/pressbooks/releases/latest/) and the [latest version of WordPress](http://codex.wordpress.org/Upgrading_WordPress)?
* [x] Are you experiencing this issue with one of our built-in root or book themes?
* [x] Did you [perform a cursory search](https://github.com/issues?q=+is%3Aissue+repo%3Apressbooks%2Fpressbooks) to see if your bug or enhancement is already listed?

### Description

An image in your chapter can be set to link to its own media file page, which allows readers to open the image in a new webpage at full size. 
![Screen Shot 2019-11-14 at 3 50 40 PM](https://user-images.githubusercontent.com/39064028/68895030-83bacb80-06f6-11ea-9fd5-a3e7e1756fb8.png)

When a book is cloned, images which are set to link to their own media files will still link to the **source** book/network media file instead of linking to the image file in the new book on the new network. 

The **Link to** setting is automatically changed to "Custom URL" 
![Screen Shot 2019-11-14 at 3 57 59 PM](https://user-images.githubusercontent.com/39064028/68895557-8964e100-06f7-11ea-8e04-4315c989b446.png)

Example: https://linktomediaclone.textopress.com/chapter/chapter-1/ was cloned from 
Original: https://integrations.pressbooks.network/admintestbook/

### Steps to Reproduce

1. Create a chapter and add an image
2. In the Image Display Settings, set the image to link to "Media File" then save
3. Make sure the book is public and openly licensed
4. Clone the book to another network
5. View the media file that the image in the cloned book links to 

**Expected behavior:** Images set to link to their media file will automatically switch to link to the image file in the new book when cloned

**Actual behavior:** Images in cloned books will still link to source book media files.

**Comment by SteelWagstaff** [comment]
@tw77 @MichSpieler not urgent, but curious to know if this is still a relevant bug/issue in latest release of Pressbooks?

**Comment by tw77** [comment]
Yes, this bug/issue is still present in latest release of Pressbooks.

**Comment by arzola** [comment]
This issue has been automatically closed as part of an organization-wide backlog review.

**Reason:** This issue was created 6y 3m ago with no updates in the last 5m.

If this is still relevant, please feel free to reopen it with updated context. We appreciate your contributions and want to ensure our issue tracker stays actionable.

**Learn more:** https://pressbooks.community/t/announcement-automatic-closure-of-old-github-tickets/4542
